frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -24,11 +34,77 @@ const navigateTo = (e: Event, url: string) => {
   e.preventDefault();
 };
 
+const resourceAttributes = (
+  resource: string,
```

Comment:
Consider taking in the model, which has both the resource and group

Reply:
@spadgett Updated this function.